recipes/dvc-objects/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "dvc-objects" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 65370e3ea83b4c4b2ad8567ef8aa79b5b4c0fc2b3a4ff6ea2290ca64a65c6fcf
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
```

Comment:
Upstream also lists `jaraco.windows`. Since you're also upstream, is that no required?

Reply:
@BastianZim Nice catch! I've updated the PR: dropped python 3.7 so we don't even need `jaraco.windows` anymore. Part of the upcoming 3.7 drop in DVC https://github.com/iterative/dvc/issues/7708 . Thank you 🙏